pkg/services/contexthandler/contexthandler.go

```diff
@@ -127,6 +131,16 @@ func (h *ContextHandler) Middleware(mContext *web.Context) {
 		ol.Int64("orgId", reqContext.OrgId),
 		ol.Int64("userId", reqContext.UserId))
 
+	if !h.ac.IsDisabled() {
+		permissions, err := h.ac.GetUserPermissions(reqContext.Req.Context(), reqContext.SignedInUser)
+		if err != nil {
+			reqContext.JsonApiErr(403, "failed to get permissions", err)
+			return
+		}
+		mContext.Req = mContext.Req.WithContext(accesscontrol.ContextWithPermissions(mContext.Req.Context(), permissions))
```

Comment:
First idea was to add permissions to the SignedInUser type but that would create a circular import (models -> accesscontrol -> models).